operations/nhc.rst

```diff
@@ -91,6 +100,15 @@ states to perform the checks:
    $ juju config slurmd health-check-interval=300
    $ juju config slurmd health-check-state="CYCLE,ANY"
 
+.. note::
+
+   NHC does not *undrain* a node. If a node was drained and NHC runs on that
+   node, the node will continue on the drained state, regardless of the checks
+   passing or failing.
```

Comment:
```suggestion
   NHC does not *undrain* a node. If NHC checks a drained node that no longer has disk issues,
   the node will remain in the drained state, regardless of the checks passing or failing.
```

Reply:
Hmmm I'm not sure about this... NHC goes way beyond disk issues.. I'll come back at this tomorrow morning.